Tasks/AzureWebPowerShellDeployment/Publish-AzureWebDeployment.ps1

```diff
@@ -41,6 +41,8 @@ Write-Host "Package= $Package"
 Write-Host "AdditionalArguments= $AdditionalArguments"
 
 #Find the package to deploy
+import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
```

Comment:
the task was working because a previous task import the task.common module, it cached in memory, so this task works fine, but if this task was the one that run first, it will cause problem.